signals.mss

```diff
@@ -1325,4 +1348,167 @@ Format details:
       }
     }
   }
+
+  /*******************************************************/
+  /* PL main, combined and distant light signals */
+  /*******************************************************/
+  ["feature"="PL-PKP:s"] {
```

Comment:
This is nothing that is up to you, but I wonder if we should factor out the country specific signals to their own files and only `#include` them here (however that works in CartoCSS). @Nakaner?